crates/sui-core/src/signature_mpc/sign.rs

```diff
@@ -92,23 +103,22 @@ impl SignRound {
     pub(crate) fn complete_round(&mut self, state: SignState) -> Result<SignRoundCompletion> {
         let round = mem::take(self);
         match round {
-            // todo(zeev): remove this enum and handle the `mem` code.
+            // TODO (#131): Remove redundant enum and handle mem call.
```

Comment:
This (`TODO (#[ISSUE])`) is a really good practice @zeevmoney

Reply:
I don't like Todos in the code, they are in there only to be removed while doing the CR.